1.0/docs/start/first-element/step-4.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: default
+type: start
+shortname: Start
+title: "Step 4: React to input"
+subtitle: Build your first Polymer element
+---
+
+
+Of course, a button isn't a button if you can't click it. To toggle the button,
+add an event listener. To add event listeners on the host element (in this
+case, `icon-toggle`), add a `listeners` object to the element prototype:
+
+#### icon-toggle.html
+{: .caption }
+
+
+<pre class="prettyprint">
+  Polymer({
+    /* this is the element's prototype */
+    is: 'icon-toggle',
+    properties: {
+      toggleIcon: String,
+      pressed: {
+        type: Boolean,
+        value: false,
+        notify: true,
+        reflectToAttribute: true
+      }
+    },
+    listeners: {
+      'tap': 'toggle'
+    },
+    toggle: function() {
+      this.pressed = !this.pressed;
+    },
+  });
+</pre>
+
+Key information:
+
+*   The [`listeners object`](https://www.polymer-project.org/1.0/docs/devguide/events.html#event-listeners)
+    maps event names to handler names.
+
+*   The `tap` event is generated by Polymer's [gesture system](https://www.polymer-project.org/1.0/docs/devguide/events.html#gestures)
```

Comment:
I was thinking it might be good if it also mentions that you're not limited to listening for polymer events, you can also listen for `keydown`, etc. inside of `listeners`.